css/normalize.css

```diff
@@ -0,0 +1,424 @@
+/*! normalize.css v4.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Personally no, the one you have looks fine, just wanted to make a note to prevent possible future head-banging.